test/Tester/EventSourcingTests/CountersGrainPerfTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections.Concurrent;
+using System.Linq;
+using Orleans;
+using TestGrainInterfaces;
+using Xunit;
+using Assert = Xunit.Assert;
+using TestExtensions;
+using Xunit.Abstractions;
+using Orleans.Runtime;
+using System.Collections.Generic;
+using Xunit.Sdk;
+
+namespace Tester.EventSourcingTests
+{
+    [TestCaseOrderer("SimplePriorityOrderer", "Tester")]
+    public partial class CountersGrainTests
```

Comment:
I just run them all from within visual studio and look at the time reported for each. It's not very precise but gives a rough impression.